### ISSUE TITLE
Differentiate between tag and valuie

### ISSUE THREAD

**Issue by Bobronium** [leonhard-s/python-sphinx-highlighter#7]
## Problem
I'm porting Darcula theme from PyCharm and trying to achieve same look for docstrings.

Your extension works like a charm, however it doesn't differentiate between tag and its value, making it impossible to replicate PyCharm look, or do other customisations (e.g. highlight only param value, or highlight it s source):
 <table>
  <tr>
    <th>PyCharm</th>
    <th>VSCode</th>
  </tr>
  <tr>
    <td>
<img width="494" alt="image" src="https://user-images.githubusercontent.com/36469655/168847188-c4f6ad72-f6b0-4128-b49c-6f212e9c7ffb.png">
</td>
    <td>
<img width="495" alt="image" src="https://user-images.githubusercontent.com/36469655/168847702-3a543838-9729-4e0a-aa6f-7682cc8643a2.png">
</td>
  </tr>
</table>

<details>
  <summary>Details</summary>

#### My current json settings:
```js
        {
            "scope": [
                "entity.name.tag.sphinx",
                "punctuation.definition.interpreted.sphinx"
            ],
            "settings": {
                "foreground": "#629755",
                "fontStyle": "italic bold underline"
            }
        },
```
#### Inspector:
<img width="795" alt="image" src="https://user-images.githubusercontent.com/36469655/168849219-85cf469a-99fb-4f85-8c23-029a02f452a2.png">  
  
</details>

## Proposal
Introduce separate scope for tag value, e.g. `entity.name.tag.value`

Thank you for the awesome extension!

**Comment by leonhard-s** [comment]
Both VS Code and TextMate allegedly support named RegEx groups, but I have been unable to make them work in VS Code grammar definitions, so I will have to keep using indices to identify capture groups.

I have introduced the `entity.name.tag.value.sphinx` scope to the name portion of a named tag, with `entity.name.tag.sphinx` still being used for unnamed tags, the tag identifier itself, and the domain selector for cross-references:

<details>
<summary>Example screenshots from Development version</summary><br>

![image](https://user-images.githubusercontent.com/35902139/168905016-c2c41da0-7781-44ac-9377-af47d2c6db46.png)

![image](https://user-images.githubusercontent.com/35902139/168905203-6c43d084-28f7-4c53-aee5-1610078d1748.png)

![image](https://user-images.githubusercontent.com/35902139/168905249-078a7110-1670-408f-bc13-130c28514be9.png)

</details>

I have pushed the above changes to the [`develop`](https://github.com/leonhard-s/python-sphinx-highlighter/tree/develop) branch if you wish to do further testing.

Please do let me know if you encounter any other issues with the current parsing behaviour. I originally created this extension to help me navigate my own projects, and scope name selection was therefore often governed by what provided good contrast in the themes I used most.

**Comment by leonhard-s** [comment]
Closing as this was implemented in fe1d69b0.

**Comment by Bobronium** [comment]
Thank you! Any ETA on releasing the update to marketplace?

**Comment by Bobronium** [comment]
Thank you! Any ETA on releasing the update to marketplace?

**Comment by leonhard-s** [comment]
I just published v0.2.0 on the VS Code Marketplace containing the new additions. It might take a few hours before it shows up due to caching, but it should become available to you over the course of the day.

Thank you for the contributions!

**Comment by Bobronium** [comment]
Thank you for fast response and great extension!
